readme.md

```diff
@@ -20,29 +22,28 @@ after forking or cloning this repository
 
 1. make sure you are using node >=12. `node --version`
 2. run `$ yarn` this should install the dependencies with yarn 2 (berry).
-  **you must use yarn 2**
+  **You must use yarn 2** [migrating from yarn 1](https://yarnpkg.com/getting-started/install)
```

Comment:
I think this might be misleading, partly my fault. IIRC you just need yarn 1.2+ installed. you shouldn't have to do these migration steps since the repo has already done them. yarn (1) will see that you have a `yarnrc.yml` and use the yarn (2) bin that's been included in the repo

Reply:
Thanks for the explanation that makes perfect sense @jenbutongit now that I've looked in the .yarnrc.yml

Reply:
@TheSpartan1980 please pull master because I had changed that part of the documentation in the webpack PR that just got merged.